bundles/persistence/org.openhab.persistence.sitewhere/.classpath

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="src/main/java"/>
+	<classpathentry kind="src" path="src/main/resources"/>
+	<classpathentry exported="true" kind="lib" path="lib/spring-beans-3.2.10.RELEASE.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/spring-context-3.2.10.RELEASE.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/spring-context-support-3.2.10.RELEASE.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/spring-core-3.2.10.RELEASE.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/spring-web-3.2.10.RELEASE.jar"/>
```

Comment:
i assume spring is a transitive dependency of the sitewhere agent? Isn't there a spring-all bundle available (size?)